runtime/reconcilers/explore.go

```diff
@@ -201,3 +215,48 @@ func (r *ExploreReconciler) resolveFields(selected []string, selector *runtimev1
 	}
 	return res, nil
 }
+
+// Merges a list of security condition rules into a single rule.
+func mergeConditionRules(rules []*runtimev1.SecurityRule) *runtimev1.SecurityRule {
```

Comment:
It only merges `Access` rules, not `FieldAccess` or `RowFilter` or any other – that should probably be made clearer in the docstring and/or function name here